### DISCUSSION TITLE
treesitter giving error all of sudden

### DISCUSSION THREAD

**Discussion by Kim-Robin** [LazyVim/LazyVim#900] [Q&A] [answered]
My lazyvim setting was working fine for while and all of sudden I got error message related to treesitter highlighting on comment. I did run checkhealth to see what is going on and treesitter is showing error on comment marked X on highlighting.
Does anyone know how to fix this issue?

**Comment by amaanq** [comment]
Either `:TSUpdate` or same thing here:
https://github.com/nvim-treesitter/nvim-treesitter/pull/4903

**Reply by Kim-Robin** [reply]
I have checked the link that you have shared. Where can I find " ((uri) @text.uri @nospell)" and remove to make it work?

**Reply by amaanq** [reply]
wherever nvim-ts is installed, and either way that's not the issue, the issue is the snap package bundling the comment parsers (and others?)

**Reply by Kim-Robin** [reply]
sorry I am kind of new to the nvim. Is there way to disable bundling the comment parser on snap package?

**Reply by amaanq** [reply, marked as answer]
my recommendation is to install it from another source, or just build it yourself, I have no idea why they're bundling the parser and they need to stop
